modules/configuration/src/main/java/org/apache/ignite/configuration/presentation/json/JsonConverter.java

```diff
@@ -43,6 +44,15 @@
         return gson.toJson(res);
     }
 
+    /** {@inheritDoc} */
+    @Override public String rootName(String source) {
```

Comment:
This method assumes that there's only one root, right?

Reply:
Yes, it cannot work with an update request packing more than one root. We may reconsider and change this in the future.